docs/9_0/components/datatable.md

```diff
@@ -43,7 +43,7 @@ DataTable displays data in tabular format.
 | expandedRow               | false              | Boolean          | Defines if row should be rendered as expanded by default.
 | filterBy                  | null               | Map              | Map of filters; This also allows to filter the table by default.
 | filterDelay               | 300                | Integer          | Delay in milliseconds before sending an ajax filter query.
-| filterEvent               | keyup              | String           | Event to invoke filtering for input filters.
+| onFilter                  | keyup              | String           | Event to invoke filtering for input filters.
```

Comment:
Why change the `onFilter` its been `filterEvent` for years is it really worth this breaking change?  Also anything in our other components labeled `on` typically means it is a client side JS hook and that is not the case here.

Reply:
Yup my bad, I thought it was a client side callback